### ISSUE TITLE
random_noise is not defined 

### ISSUE THREAD

**Issue by yuki4ghub15** [fhtanaka/directed_research_CS_2018#9]
Hello tanaka, 

How do  you define random_noise here.  
>>fake_data = generator(random_noise(real_data.size(0))).detach()
